JesusFilm/core
# fix: eng 987 empty search doesnt reset videos in see all videos

# Description

### Issue

On delete of last character, the titles input didn't refine by empty string. The formik doesn't seem to be firing the onChange/onSubmit event.

[Link to Linear Todo](https://linear.app/jesus-film-project/issue/ENG-987/empty-search-doesnt-reset-videos-in-see-all-videos)

### Solution

Manually handling change to title outside the Formik. 

### External Changes

Refactored the FilterList spec file

### How to test
- when text from titles deleted with backspace, should return return empty search resources and remove query param from url
- when text from titles deleted with X button, should return return empty search resources and remove query param from url
